repos/PredictionIO/core/src/main/scala/io/prediction/controller/Metric.scala

```diff
@@ -63,7 +63,7 @@ private[prediction] trait StatsMetricHelper[EI, Q, P, A] {
         evalDataSet.map {
           case (_, qpaRDD) =>
             qpaRDD.map { case (q, p, a) => calculate(q, p, a) }
-        }
+        },
```

Comment:
The original is weird. Create a union of 1??

https://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.SparkContext@union[T](first:org.apache.spark.rdd.RDD[T],rest:org.apache.spark.rdd.RDD[T]*)(implicitevidence$7:scala.reflect.ClassTag[T]):org.apache.spark.rdd.RDD[T]